cocotb/triggers.py

```diff
@@ -455,11 +482,13 @@ def _outcome(self):
 
     @property
     def retval(self):
-        """FIXME: document"""
+        """The return value of the joined coroutine.
+
+        If the coroutine threw an exception, this attribute will re-raise it.
+        """
         return self._coroutine.retval
 
     def prime(self, callback):
-        """FIXME: document"""
```

Comment:
Dropping this documentation entirely, since it just implements the base class method